alchemist/alchemist-swingui/src/main/java/it/unibo/alchemist/boundary/monitors/package-info.java

```diff
@@ -8,5 +8,6 @@
 /**
  * This package contains generic monitors for various Environments.
  */
+@Deprecated
```

Comment:
This is not deprecated at all

Reply:
Most of the SwingUI classes are marked as deprecated in feature-fxui, should I remove the annotation from the classes as well or just from the package?

Reply:
I only mean that the very package is not deprecated in general. There could be legitimate monitors there.

Reply:
I see. Un-deprecated.